tests/test_ios_appstore_lane_identity.py

```
@@ -990,6 +993,174 @@ def test_validate_appstore_release_prepares_content_rights_and_build(
+        _check(
+            len(included) == 3
+            and all(
+                isinstance(item, dict)
+                and item.get("attributes", {}).get("available") is True
+                and item.get("attributes", {}).get("preOrderEnabled") is False
+                for item in included
+            ),
+            "every territory is immediately available without pre-order",
+        )
+
+        second = subprocess.run(command, env=env, capture_output=True, text=True, check=False)
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)